nginx/certificates/transcendance.key

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Isn't it a security failure to have our private key pushed in the repo ? Shouldn't it be in our .env ?

Reply:
Its a self signed certificate, so its not technically very bad to have it for development, but yes, when we get a proper certificate we should encrypt it with blackbox